cypress/e2e/search.cy.ts

```diff
@@ -0,0 +1,30 @@
+import cypress from 'cypress'
+
+describe("Search panel", () => {
+  it("should search for Test123123 and return no results", () => {
+    cy.visit("http://localhost:3000");
+    cy.url().should('be.equal', "http://localhost:3000/search");
+    cy.wait(500);
```

Comment:
Do not use cy.wait() since 500ms is very relative (on your computer it will be sufficient on another computer (CI/CD e.g.) with only one 2 core CPU it  will not. There are other ways how to wait properly https://filiphric.com/waiting-in-cypress-and-how-to-avoid-it?utm_campaign=Software%2BTesting%2BWeekly&utm_medium=email&utm_source=Software_Testing_Weekly_106